src/components/nextSection.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from 'styled-components';
+import image from './primaryImage.png'
+
+const Body = styled.a`
+    position: absolute;
+    left: 679px;
+    top: 1593px;
```

Comment:
Do not use position absolute. It's very difficult to keep the component responsive this way. You never know the screen size you're working with. Instead, you can make a separate wrapper div that "wraps" around your component that is the width of the page. Use ```display: flex``` to do this so the wrapper resizes as the screen resizes. Then, here you can use ```margin-left: auto``` on your body div to have it right-justified within the wrapper div. Also, you can count on there being other components above it that will push it to the end of the page. So no need to force it to the bottom of the page. The mockups use that type of styling because they are completely static, but we want things to be responsive in the actual product.
```suggestion
    margin-left: auto;
```